assets/csidriveroperators/openstack-manila/hypershift/guest/monitoring_role.patch.yaml

```diff
@@ -0,0 +1,13 @@
+- op: "add"
```

Comment:
How is this monitoring role different from the one we have in standalone? Should we move this to the base assets?

Reply:
If we add it to the base asset, it would also be added to the management assets once they are generated and that is not necessary.
Other drivers also do this, for example: [azure](https://github.com/openshift/cluster-storage-operator/blob/master/assets/csidriveroperators/azure-disk/hypershift/guest/monitoring_role.patch.yaml).